ui/client/components/graph/TypeSuggester.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+import ProcessUtils from "../../common/ProcessUtils";
+
+export default class TypeSuggester {
+
+  constructor(typesInformation){
+    this._typesInformation = typesInformation;
```

Comment:
Don't use underscore coding style.. _typesInformation replace by this.typesInformation

Reply:
done